clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -568,7 +563,7 @@ final void fail(long now, Throwable throwable) {
                 tries++;
                 if (log.isDebugEnabled()) {
                     log.debug("{} aborted at {} after {} attempt(s)", this, now, tries,
-                        new Exception(prettyPrintException(throwable)));
+                            new Exception(prettyPrintException(throwable)));
```

Comment:
nits: are the indentation changes here (and below) consistent with expected code style?